documentation/MeetingMinutes/MoM-2023-11-21.md

```diff
@@ -19,8 +32,18 @@ Device Location minutes: [https://github.com/camaraproject/DeviceLocation/tree/m
 
 * Review latest comments in ongoing PRs for APIs, and decide if ready to be merged:
   - [location-verification.yaml v0.2 Release Candidate](https://github.com/camaraproject/DeviceLocation/pull/104)
+       - Point about Ipv6 still open in QoD (will be valid for all three APIs)
+       - Description comment from Cetin to be considered
   - [location-retrieval.yaml v0.1 Release Candidate](https://github.com/camaraproject/DeviceLocation/pull/114)
+      - Error: Status & Code are 'normative' - message are open to modification by implementation
   - [geofencing.yaml v0.1 Release Candidate](https://github.com/camaraproject/DeviceLocation/pull/116)
+      - Rafal proposed to align Notification source example list accordingly to the guidelines
+
+We're still in line to release before the end of the year.
+    Not rush to close these PR

```

Comment:
My understanding was that we wanted these PRs to be merged as "release candidates", in the following days, as soon as current pending comments are resolve. This is the first step prior to make the release

```suggestion
    - Resolve pending comments and merge current PRs as -rc
    - Open testing and feedback period for RC versions

```